Lib/asyncio/tasks.py

```diff
@@ -252,6 +253,12 @@ def _wakeup(self, future):
 
 
 _PyTask = Task
+_register_task = _py_register_task = base_tasks._register_task
+_enter_task = _py_enter_task = base_tasks._enter_task
+_leave_task = _py_leave_task = base_tasks._leave_task
+_unregister_task = base_tasks._unregister_task
+all_tasks = base_tasks.all_tasks
+current_task = base_tasks.current_task
```

Comment:
Don't do 'all_tasks = base_tasks.all_tasks'. Just import all_tasks from `base_tasks` in `asyncio.__init__` and use `base_tasks.all_tasks()` in this file.